src/main/resources/templates/application/summary/proceedings-and-costs-section.html

```diff
@@ -0,0 +1,163 @@
+          <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Matter type</th>
+            <th scope="col" class="govuk-table__header">Proceeding</th>
+            <th scope="col" class="govuk-table__header">Form of civil legal service</th>
+            <th scope="col" class="govuk-table__header">Client involvement type</th>
+            <th scope="col" class="govuk-table__header">Lead</th>
+            <th scope="col" class="govuk-table__header">Action</th>
+          </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+          <tr class="govuk-table__row" th:each="proceeding : ${proceedings.content}" th:unless="${proceedings.content.isEmpty()}">
+            <td class="govuk-table__cell">
+              <a th:href="@{/application/summary/proceedings/{id}/confirm(id=${proceeding.id})}"
+                 class="govuk-link" th:text="${proceeding.matterType.displayValue}"></a>
+            </td>
+            <td class="govuk-table__cell" th:text="${proceeding.proceedingType.displayValue}"></td>
+            <td class="govuk-table__cell" th:text="${proceeding.levelOfService.displayValue}"></td>
+            <td class="govuk-table__cell" th:text="${proceeding.clientInvolvement.displayValue}"></td>
+
+            <th:block th:if="${proceeding.leadProceedingInd}">
+              <td class="govuk-table__cell">
+                <img class="govuk-!-padding-left-2 govuk-!-padding-right-2" src="/tick.gif" alt="tick">
+              </td>
+              <td class="govuk-table__cell"></td>
+            </th:block>
+
+            <th:block th:unless="${proceeding.leadProceedingInd}">
+              <td class="govuk-table__cell"></td>
+              <td class="govuk-table__cell">
+                <a th:href="@{/application/summary/proceedings/{id}/remove(id=${proceeding.id})}"
+                   class="govuk-link" th:text="${'Delete'}">
+                </a>
+                |
+                <a th:href="@{/application/summary/proceedings/{id}/make-lead(id=${proceeding.id})}"
+                   class="govuk-link" th:text="${'Make lead'}">
+                </a>
+              </td>
+            </th:block>
+          </tr>
+          <tr class="govuk-table__row" th:if="${proceedings.content.isEmpty()}">
+            <td class="govuk-table__cell" colspan="6">None</td>
+          </tr>
+          </tbody>
+        </table>
+        <a class="govuk-button govuk-button--secondary"
+           data-module="govuk-button"
+           href="/todo">
+          Add proceeding
+        </a>
+      </div>
+      <div class="govuk-tabs__panel govuk-tabs__panel--hidden" id="case-costs" aria-labelledby="tab_case-costs">
+        <h2 class="govuk-heading-l">Case costs</h2>
+        <table class="govuk-table">
+          <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Case cots</th>
```

Comment:
I know some cases deal with children. But I think you meant 'costs' :-)

Reply:
amended